string text_path
string language
string wave_path
---
bool ok
